runtime/lua/vim/lsp/inlay_hint.lua

```diff
@@ -98,6 +98,91 @@ function M.on_refresh(err, _, ctx, _)
   return vim.NIL
 end
 
+--- @class InlayHintOptions
+--- @field bufnr number|nil
+--- @field filter function|nil
+--- @field id number|nil
+--- @field name string|nil
+--- @field range { start: number[], end: number[] }|nil
+
+--- Returns inlay hints in the (optionally) given buffer, client, and range.
+--- This method does not make an LSP request, but just queries locally stored inlay hints.
+---
+--- @param options table|nil Optional table which holds the following optional fields:
```

Comment:
Why not use the options type here? That what users can get better editor support.

Reply:
I coulda sworn it broke CI before, but it seems fine now, so I've done so!